flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/decorators/InitTaskManagerDecoratorTest.java

```diff
@@ -187,6 +189,11 @@ public void testPodAnnotations() {
 		assertThat(resultAnnotations, is(equalTo(ANNOTATIONS)));
 	}
 
+	@Test
+	public void testPodServiceAccountName() {
+		assertEquals(SERVICE_ACCOUNT_NAME, this.resultPod.getSpec().getServiceAccountName());
```

Comment:
```suggestion
		assertThat(SERVICE_ACCOUNT_NAME, is(this.resultPod.getSpec().getServiceAccountName()))
```

In Flink, we prefer `assertThat` for assertion.